mixins/test/localize-mixin.test.js

```diff
@@ -269,6 +300,53 @@ describe('LocalizeMixin', () => {
 
 	});
 
+	describe('browser language settings in dynamic mixin', () => {
+
+		const browserLangsFixture = `<${browserLangsTag}></${browserLangsTag}>`;
```

Comment:
These are sort of oddball tests, but `navigator` isn't able to be stubbed which causes some timing weirdness.